src/pages/Post.js

```diff
@@ -517,6 +522,39 @@ class PostPage extends React.Component {
       endDate: endDate
     })
   }
+  
+  showFilters() {
+    if (this.state.filterToggle == false) {
+      document.getElementById("yearBoxes").style.display = "block"
+      document.getElementById("schoolBoxes").style.display = "block"
+      this.setState({
+        filterToggle: true,
+        filterToggleText: "Remove Filters",
+        filterToggleColor: "#a32512"
+      })
+    } else {
+      document.getElementById("yearBoxes").style.display = "none"
+      document.getElementById("schoolBoxes").style.display = "none"
+      this.setState({
+        filterToggle: false,
+        filterToggleText: "Add Filters",
+        filterToggleColor: "#12a340"
+      })
+    }
+  }
+
+  openModal() {
+    this.setState({modalIsOpen: true})
+  }
+
+  afterOpenModal() {
+    this.setState({modalIsOpen: this.state.modalIsOpen})
+  }
```

Comment:
What is the purpose of this? It doesn't seem to do anything.

Reply:
See #9